src/rubysys/string.rs

```diff
@@ -1,14 +1,8 @@
-use std::mem;
-
-use crate::rubysys::{
-    constant::{
-        FL_USER_1, FL_USER_17, FL_USER_2, FL_USER_3, FL_USER_4, FL_USER_5, FL_USER_6, FL_USER_7,
-        FL_USHIFT,
-    },
-    libc::size_t,
-    types::{c_char, c_long, CallbackPtr, EncodingType, InternalValue, RBasic, Value},
+use super::{
+    constant::FL_USER_7,
+    types::{c_char, c_long, EncodingType, InternalValue, Value},
 };
-
+use rb_sys::{RString, VALUE};
```

Comment:
@ianks what are the warnings for RString suggesting we do for rb_sys?

> warning: use of deprecated struct `rb_sys::RString`: To improve API stability with ruby-head, direct usage of Ruby internal structs has been deprecated. To migrate, please replace the usage of this internal struct with its counterpart in the `rb_sys::stable` module. For example, instead of `use rb_sys::rb_sys__Opaque__ExampleStruct;`, use `use rb_sys::stable::ExampleStruct;`. If you need to access the internals of these items, you can use the provided `rb-sys::macros` instead.